modelscope/models/cv/self_supervised_depth_completion/self_supervised_depth_completion.py

```diff
@@ -0,0 +1,396 @@
+from modelscope.models.cv.self_supervised_depth_completion import helper
+from modelscope.models.cv.self_supervised_depth_completion import criteria
+from modelscope.models.cv.self_supervised_depth_completion.metrics import AverageMeter, Result
+# import mmcv
+from argparse import ArgumentParser
+# import torchvision
+from os import makedirs
+from tqdm import tqdm
+import cv2
+import numpy as np
+import torch.utils.data
+import torch.optim
+import torch.nn.parallel
+import torch
+import time
+# import argparse
+import os
+import sys
+os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
+from modelscope.utils.logger import get_logger
+from modelscope.utils.constant import Tasks
+from modelscope.models.builder import MODELS
+from modelscope.models.base.base_torch_model import TorchModel
+from modelscope.metainfo import Models
+from modelscope.models.cv.self_supervised_depth_completion.inverse_warp import Intrinsics, homography_from
+from modelscope.models.cv.self_supervised_depth_completion.model import DepthCompletionNet
+from .dataloaders.kitti_loader import load_calib, oheight, owidth, input_options, KittiDepth
+
+
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+
+
+# from modelscope.utils.config import Config
+
+
+m_logger = get_logger()
+
+
+@MODELS.register_module(
+    Tasks.self_supervised_depth_completion, module_name=Models.self_supervised_depth_completion)
+class SelfSupervisedDepthCompletion(TorchModel):
+    """SelfSupervisedDepthCompletion Class"""
+    def __init__(self, model_dir: str, *args, **kwargs):
```

Comment:
model is expected to be a subclass of TorchModel which should have __init__ func to initiate the network of model, and __call__ function which implements the forward process of the model.  it seems this model implementation is more like a trainer